pkgs/applications/audio/noise-repellent/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, meson, ninja, pkgconfig, fftwFloat, lv2 }:
+
+stdenv.mkDerivation rec {
+  pname = "noise-repellent";
+  version = "unstable-29-12-2018";
```

the date format is YYYY-MM-DD https://nixos.org/nixpkgs/manual/#sec-package-naming
```suggestion
  version = "unstable-29-12-2018";
```

---

Thanks for the feedback.  Done.